README.md

```diff
@@ -0,0 +1,65 @@
+# Sensu Go Ansible Collection
+
+Sensu Go [Ansible Collection][collection] contains:
+
+* Ansible role to install [Sensu Go][sensu] on your nodes.
+* Ansible modules for interacting with [Sensu Go][sensu] REST API.
+
+   [collection]: https://docs.ansible.com/ansible/latest/user_guide/collections_using.html
+                 (Using Ansible Collections)
+   [sensu]: https://docs.sensu.io/sensu-go/latest
+            (Sensu Go documentation)
+
+
+## Installation
+
+If we would like to install the Sensu Go collection, we need to have Ansible
+2.9 or later installed. Once we have taken care of this, we can install the
+Sensu Go collection from [Ansible Galaxy][galaxy] by running:
+
+    $ ansible-galaxy collection install sensu.sensu_go
+
+   [galaxy]: galaxy.ansible.com (Ansible Galaxy)
+
+
+## Usage
+
+You can use the namespaced modules:
```

Comment:
These 2 ways (namespaced vs. bare names) of referencing the collection from playbooks are valid for roles too, right? If yes, then I think it wouldn't hurt to mention that this applies to roles too, just in case :)